backend/src/main/java/ca/bc/gov/hlth/hnweb/config/WebClientConfig.java

```diff
@@ -142,8 +136,6 @@ public WebClient hibcWebClient() throws HNWebException {
                 .filter(logRequest())
                 .defaultHeader(HttpHeaders.CONTENT_TYPE, MediaType.TEXT_PLAIN_VALUE) 
                 .defaultHeader(HttpHeaders.ACCEPT, MediaType.ALL_VALUE)
-                // TODO (weskubo-cgi) Remove default headers and corresponding properties
```

Comment:
Doesn't the R15 still require basic Auth? 
Also don't the R15 and R50 share an endpoint? Should these just be the same WebClient?

Reply:
We discovered during testing that E45/R15 and R50 are actually set up as different user accounts for basic auth. So to support this the auth was decoupled from the webclient. I established the pattern in E45/R15 and Dave will update the code so there isn't an R50 specific webclient or service methods...but just it's own set of credentials.